src/Sparklines.js

```diff
@@ -56,7 +56,8 @@ class Sparklines extends React.Component {
         return (
             <svg {...svgOpts} >
                 {React.Children.map(this.props.children, child =>
-                    React.cloneElement(child, { points, width, height, margin })
+                    React.cloneElement(child, { points, limit, width, height, margin, max: typeof max == "undefined" ? Math.max.apply(Math, data) : max,
```

Comment:
Why does the min and max need to be calculated here? The Sparklines component doesn't do any additional calculations and the dataToPoints function backfills the missing information. The second commit seems extraneous.